partcad/src/partcad/assembly_factory_file.py

```
@@ -41,7 +41,20 @@ def post_create(self) -> None:
             pc_logging.warning(f"Assembly path is not set: {self.assembly.name}")
         super().post_create()
 
-    async def instantiate(self, assembly):
+    async def download_file_async(self, assembly) -> None:
+        """Fetch what 'fileFrom' points at, unless the file is already there."""
         if not self.fileFactory is None and not os.path.exists(assembly.path):
```

Ruff (0.16.1)

[error] Test for object identity should be `is not`

Convert to `is not`

(E714)